app/viewmodels/govuk/BackLinkFluency.scala

```diff
@@ -12,14 +28,14 @@ trait BackLinkFluency {
 
```

Comment:
@iservport This seems bit different, do we need these classes BackLinkFluency etc